index.js

```diff
@@ -21,6 +21,10 @@ if (!heimdall.hasMonitor('async-disk-cache')) {
   heimdall.registerMonitor('async-disk-cache', function AsyncDiskCacheSchema() {});
 }
 
+var homedir = require('os').homedir().split(path.sep);
+var username = homedir[homedir.length - 1];
```

Comment:
well, fuckit good guy @sindresorhus  making things nice for us https://www.npmjs.com/package/username